pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -50,6 +50,13 @@ message ApplicationConfiguration {
 
   // Common configuration options that apply to all Buildbarn binaries.
   buildbarn.configuration.global.Configuration global = 19;
+
+  // Install devices into /dev of the input root. Useful in combination
+  // with having the runner chroot into the input root.
+  bool install_devices = 20;
+
+  // Device nodes to install in input root
```

Comment:
(Nit: End all sentences with a full stop.)

This should likely contain some example, right? Otherwise it becomes ambiguous whether the /dev/ prefix needs to be included.

Also, this is currently configured at the global level, even though I think it would make far more sense to be able to specify this at the runner level. I can well imagine a multi-purpose worker that does chrooting for a certain runner, while doing plain Ubuntu container builds on another runner.